ios/ios_testing_references.md

```diff
@@ -68,9 +76,50 @@ class SearchViewModel {
 
 In the above example, the `SearchViewModel` can be initialized with a `NetworkingLayer` to allow a mock `NetworkingLayer` to be injected into the `SearchViewModel`. In this example `NetworkingLayer` is a protocol and as a result the live and mock network layers can both conform to the protocol. The class `DefaultNetworkingLayer` conforms to `NetworkingLayer` and is included as a default initializer. Using protocol is an effective way to implement dependency injection.
 
+### Mocking
+
+As eluded to in the previous section, mocking is a process of creating a dummy or 'mock' object, which will simulate the behavior of a real dependency. This is useful if the real objects are impractical to incorporate into a unit test. Objective-C has access to tools like OCMockito which allow you to automatically generate mock objects at runtime. Unfortunately, because of limitations in the Swift language, you are forced to write your own mocks.
```

Comment:
I would leave out this last part about Mockito.  IMHO if you are testing using protocols properly and testing via an interface it's actually an advantage of swift that you build your own, and are not dragging around a massive thing like mockito

Reply:
👍